inference/core/interfaces/http/http_api.py

```diff
@@ -340,6 +342,19 @@ def load_core_model(
         The DocTR model ID.
         """
 
+        load_grounding_dino_model = partial(
```

Comment:
Has this function been tested? I think this function is going to look at the request for a field called `grounding-dino_version_id` which isn't a field on the request object, so I think it's going to throw an error....